bika/lims/content/analysisservice.py

```diff
@@ -602,5 +601,4 @@ def after_deactivate_transition_event(self):
             template = api.get_object(template)
             template.remove_service(self)
 
-
 registerType(AnalysisService, PROJECTNAME)
```

Comment:
the blank lines can be left here, they are according to PEP8

Reply:
Done